src/Illuminate/Support/helpers.php

```diff
@@ -379,6 +379,19 @@ function collect($value = null)
     }
 }
 
+if (! function_exists('collectDepth')) {
+    /**
+     * Create a collection from a multi-level array.
+     *
+     * @param  mixed  $value
+     * @return \Illuminate\Support\Collection
+     */
+    function collectDepth($value = null)
```

Comment:
I would not create a helper function for a functionality that we do not know if is going to need a helper function.
And for helper functions we use the slug notation